src/features/projectsV2/ProjectDetails/components/PlantLocationInfo.tsx

```diff
@@ -5,19 +5,20 @@ import {
   SamplePlantLocation,
 } from '../../../common/types/plantLocation';
 import styles from '../styles/PlantLocationInfo.module.scss';
-import ImageCarousel from './ImageCarousel';
 import PlantLocationHeader from './microComponents/PlantLocationHeader';
 import SpeciesPlanted from './microComponents/SpeciesPlanted';
 import SampleTrees from './microComponents/SampleTrees';
 import TreeMapperBrand from './microComponents/TreeMapperBrand';
 import PlantingDetails from './microComponents/PlantingDetails';
 import { useTranslations } from 'next-intl';
+import ImageSlider from './microComponents/ImageSlider';
 
-const PlantLocationInfoSection = ({
-  plantLocationInfo,
-}: {
+interface Props {
   plantLocationInfo: PlantLocation | SamplePlantLocation | null;
-}) => {
+  isMobile: boolean;
+}
+
+const PlantLocationInfo = ({ plantLocationInfo, isMobile }: Props) => {
   const isMultiTreeRegistration =
```

Comment:
Since this is only for multi-tree-registrations, updates types and logic accordingly. Also, name the component MultiPlantLocationInfo